nef/Command.swift

```diff
@@ -0,0 +1,56 @@
+//  Copyright © 2020 The nef Authors.
+
+import Foundation
+import SourceEditorModels
+
+enum Command {
+    case about
+    case preferences
+    case exportSnippetToFile(selection: String)
+    case exportSnippetToClipboard(selection: String)
+    case markdownPage(playground: String)
+    case playgroundBook(package: String)
+    case notification(userInfo: [String: Any], action: String)
+}
+
+extension Command {
+    var menuKey: String {
+        switch self {
+        case .about:
+            return MenuEditorCommand.about.key
+        case .preferences:
+            return MenuEditorCommand.preferences.key
+        case .exportSnippetToFile:
+            return MenuEditorCommand.exportSnippetToFile.key
+        case .exportSnippetToClipboard:
+            return MenuEditorCommand.exportSnippetToClipboard.key
+        case .markdownPage:
+            return MenuEditorCommand.markdownPage.key
+        case .playgroundBook:
+            return MenuEditorCommand.playgroundBook.key
+        case .notification:
+            return ""
```

Comment:
What's the reason for this key to be empty?

Reply:
it is a missing key - it should be 'notification'